esm-samples/jsapi-angular-cli/README.md

```diff
@@ -1,6 +1,12 @@
-# ArcGIS JSAPI Integration
+# ArcGIS API for JavaScript with Angular CLI
 
-## ArcGIS JS API notes
+**Note:** we currently recommend setting `buildOptimizer:false` in `angular.json`. This prevents WebGL errors such as `WebGL: INVALID_OPERATION` and `WebGL: INVALID_ENUM`. It may increase your on-disk and loaded application size.
```

Comment:
Maybe change `buildOptimizer:false` to `"buildOptimizer": false`

Reply:
Thanks. Done, I also removed an extra "the".